presto-docs/src/main/sphinx/connector/cassandra.rst

```diff
@@ -252,7 +252,23 @@ Miscellaneous
 * Queries without filters containing the partition key result in fetching all partitions.
   This causes a full scan of the entire data set, therefore it's much slower compared to a similar
   query with partition key in filter.
-* ``IN`` list filters are only allowed on index columns. [question: what are "index columns"?]
-* Range (``<`` or ``>`` and ``BETWEEN``) filters can be applied only to the partition key
-  with the use of the ``token()`` function. [what's the token function?]
-* Aggregates are not pushed down.
+* ``IN`` list filters are only allowed on index (partition key or clustering key) columns.
+* Range (``<`` or ``>`` and ``BETWEEN``) filters can be applied only to the partition keys.
+  with the use of the ``token()`` function.
+* Clustering keys are not pushed down, therefore there's no performance gain from using them.
+* Aggregates are also not pushed down.
+
+Partition keys can be of the below types only:
```

Comment:
Move this to data types
Also, make Data Types a top level header, and move Limitations below -- the data type support doesn't really count as limitations